requirements.txt

```diff
@@ -0,0 +1,20 @@
+appdirs==1.4.3
```

Comment:
As you suggested, I had to change this to appdirs==1.4.2 or else I couldn't build the docker image. It told me that 1.4.2 was already installed (maybe along with a different package?). I didn't see any issues with running 1.4.2.